wrapper.js

```diff
@@ -85,8 +86,9 @@ function setupMethods (soljson) {
     supportsMulti: compileJSONMulti !== null,
     supportsImportCallback: compileJSONCallback !== null,
     // Use the given version if available.
-    useVersion: function (versionString) {
-      return setupMethods(require('./bin/soljson-' + versionString + '.js'));
+    useVersion: function (compilerRepo, versionString) {
+      versionString = versionString.replace('soljson-', '');
```

Comment:
I am just a bystander and comments reached far more than just the changes on hand for that feature. ;)